buildpipeline/pipelines.json

```diff
@@ -354,6 +354,39 @@
             "PB_BuildType": "Release"
           }
         },
+		{
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
+          "Parameters": {
+            "HelixJobType": "test/functional/cli/",
+            "TargetsWindows": "true",
+            "Rid": "win-x64",
+            "TargetQueues": "Windows.10.Amd64.Core",
+            "TestContainerSuffix": "windows",
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Windows",
+            "SubType":  "Build-Tests",
+            "Type": "build/product/",
+            "PB_BuildType": "Release"
+          }
+        },
+        {
+          "Name": "Dotnet-CoreClr-Trusted-BuildTests",
```

Comment:
Same comment here, but edit this list instead - https://github.com/dotnet/coreclr/blob/012a404300e459a5d8115beba6f2c843950bbaca/buildpipeline/pipelines.json#L346